admin/nodes/views.py

```diff
@@ -157,6 +157,7 @@ def delete(self, request, *args, **kwargs):
             if node.is_deleted:
                 node.is_deleted = False
                 node.deleted_date = None
+                node.root_folder.restore()
```

Comment:
"Restore node" functionality in the admin app now restores the node's osfstorage root folder. 

(Incidentally, this functionality broke in the admin app in the fall when we started deleting files as a part of the node deletion process.  Until that point, we were not deleting the node's root folder, so when a node was restored, it's osfstorage addon already worked.)